### PR TITLE
Add Postgres helper to calculate net funding payments between blocks

### PR DESCRIPTION
**Pull request by davidli1997**
### Changelist
Add `getNetFundingPaymentsBetweenBockHeightsForSubaccount` which calculates the net funding payments a subaccount receives in a time period.

### Test Plan
- Test funding payments of different subaccounts
- Test funding payments of difference size/sign
- Test no funding payments found

### Author/Reviewer Checklist
- [ ] If this PR has changes that result in a different app state given the same prior state and transaction list, manually add the `state-breaking` label.
- [ ] If the PR has breaking postgres changes to the indexer add the `indexer-postgres-breaking` label.
- [ ] If this PR isn't state-breaking but has changes that modify behavior in `PrepareProposal` or `ProcessProposal`, manually add the label `proposal-breaking`.
- [ ] If this PR is one of many that implement a specific feature, manually label them all `feature:[feature-name]`.
- [ ] If you wish to for mergify-bot to automatically create a PR to backport your change to a release branch, manually add the label `backport/[branch-name]`.
- [ ] Manually add any of the following labels: `refactor`, `chore`, `bug`.

## Summary by CodeRabbit

* **New Features**
  * Added the ability to calculate net funding payments for a specific subaccount within a given block height range.
* **Tests**
  * Introduced comprehensive tests to ensure accurate calculation of net funding payments, including handling of boundary cases and scenarios with no payments.

### CODE REVIEW

File: indexer/packages/postgres/src/stores/funding-payments-table.ts [hunk 1]
```diff
@@ -153,6 +158,34 @@ export async function findById(
...
+  createdBeforeOrAtHeight: string,
+  options: Options = DEFAULT_POSTGRES_OPTIONS,
+): Promise<Big> {
+  const queryString: string = `
+    SELECT SUM("payment") AS "netPayments"
+    FROM funding_payments
+    WHERE "subaccountId" = '${subaccountId}'
+      AND "createdAtHeight" > ${createdAfterHeight}
+      AND "createdAtHeight" <= ${createdBeforeOrAtHeight};
+    `;
```
[the remark this answers is not in the document]
Comment (Kefancao) [reply indexer/packages/postgres/src/stores/funding-payments-table.ts]: Should address this. I think you can do a sum with the existing baseQuery model like 

.sum(payment)
.where(...)

File: indexer/packages/postgres/src/stores/funding-payments-table.ts [hunk 2]
```diff
@@ -153,6 +158,34 @@ export async function findById(
     .returning('*');
 }
 
+export async function getNetFundingPaymentsBetweenBockHeightsForSubaccount(
+  subaccountId: string,
+  createdAfterHeight: string,
+  createdBeforeOrAtHeight: string,
+  options: Options = DEFAULT_POSTGRES_OPTIONS,
+): Promise<Big> {
+  const queryString: string = `
```
Comment (Kefancao) [inline indexer/packages/postgres/src/stores/funding-payments-table.ts]: Can use

.sum(payment)
.where(...)

**Comment by Kefancao** [comment]
https://github.com/Mergifyio backport release/indexer/v9.x
